ios/FluentUI/List/ListItem.swift

```diff
@@ -338,3 +323,22 @@ public extension ListItem where LeadingContent == EmptyView {
         self.tokenSet = ListItemTokenSet(customViewSize: { layoutType.leadingContentSize })
     }
 }
+
+/// Provide defaults for generic types so static methods can be called without needing to specify them.
+public extension ListItem where LeadingContent == EmptyView, TrailingContent == EmptyView, Title == String, Subtitle == String, Footer == String {
```

Comment:
I think in a `guard` or `if` statement, the `,` works sort of like an `&&`. Is that also true for `where`? Or, rather, does this still work when any one of the properties is specified?

Reply:
yup! it works like a && in this scenario also.

Reply:
If you see the other extensions defined in this file, you can see I have to do a lot of different variations so that you only need to specify a subset. So if there is another generic content added, it makes the matrix of extensions larger ha.